scout/commands/load/report.py

```diff
@@ -6,21 +6,20 @@
 LOG = logging.getLogger(__name__)
 
 
-@click.command()
+@click.command('delivery-report')
 @click.argument('case_id')
-@click.argument('analysis_date')
 @click.argument('report_path', type=click.Path(exists=True))
 @click.argument('update', required=False)
```

Comment:
Nice! What about this, here:
@click.option('-update', '--update', is_flag=True, help='update delivery report for a sample')